src/js/prepare-build-tasks.js

```diff
@@ -31,11 +32,23 @@ var context = new buildTools.BuildContext(
 var packagePath = options.packagePath;
 
 
+function html(context, opts, baseUrl, assetContext) {
+  if (!opts.pageDefs) {
+      return htmlForPageDef(context, opts.pageDef, baseUrl, assetContext);
```

Comment:
Did I understand this correctly: we either have `opts.pageDefs` as an array, or then just a single `opts.pageDef`?